server/data/index.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/first */
+/*
+ * Do appinsights first as it does some magic instrumentation work, i.e. it affects other 'require's
+ * In particular, applicationinsights automatically collects bunyan logs
+ */
+import { initialiseAppInsights } from '../utils/azureAppInsights'
+import applicationInfo from '../config'
+
+initialiseAppInsights()
+// buildAppInsightsClient(applicationInfo)
+
+import HmppsAuthClient from './hmppsAuthClient'
+import { createRedisClient } from './redisClient'
+import TokenStore from './tokenStore'
+import DpsFeComponentsClient from './dpsComponentsClient'
+
+export const dataAccess = () => ({
+  applicationInfo,
+  hmppsAuthClient: new HmppsAuthClient(new TokenStore(createRedisClient('dpsFrontendComponent'))),
+  componentApiClientBuilder: new DpsFeComponentsClient(),
+})
+
+export type DataAccess = ReturnType<typeof dataAccess>
```

Comment:
i'm not sure this is right. we already initialise the App Insights stuff elsewhere. incentives was created from an older template which did not have this file and created services quite differently. i don't know what happens when you try to setup App Insights twice – in fact you seem to have something commented out. i would 1) remove this entire file 2)  remove the `services/index.ts` file too because the hmpps auth client and user service are already made elsewhere 3) just initialise the class in `server/middleware/dpsFrontendComponents.ts`